conftest.py

```diff
@@ -22,3 +14,33 @@ def mock_inline_css(monkeypatch):
     """
     import amo.helpers
     monkeypatch.setattr(amo.helpers, 'is_external', lambda css: True)
+
+
+def prefix_indexes(config):
+    """Prefix all ES index names and cache keys with `test_` and, if running
+    under xdist, the ID of the current slave."""
+
+    if hasattr(config, 'slaveinput'):
```

Comment:
Can you add a comment about that? Explaining that it means we're running under xdist?